charms/kubernetes_snaps.py

```diff
@@ -286,10 +333,13 @@ def configure_kube_proxy(
         f.write("# Generated by kubernetes-common library, do not edit\n")
         yaml.dump(kube_proxy_config, f)
 
+    config_files = {
+        kube_proxy_opts["config"],
+        kubeconfig,
+    }
```

Comment:
these are config files that may have been overwritten during the reconcile period for the kube-proxy service